definition/IOmnichannelVoipServiceResult.ts

```diff
@@ -1,9 +1,15 @@
+import { IVoipRoom } from './IRoom';
+
 export interface IAgentExtensionMap {
 	_id: string;
 	agentName: string;
 	extension: string;
 }
 
+export interface IRoomCreationResponse {
+	newRoom: boolean;
+	room: IVoipRoom;
+}
 export interface IOmnichannelVoipServiceResult {
-	result: string[] | IAgentExtensionMap[];
+	result: string[] | IAgentExtensionMap[] | IRoomCreationResponse | IVoipRoom | boolean | null;
```

Comment:
I'm afraid this wrong you should have one type for each result

Reply:
Fixed.